### PR TITLE
An experiment to see if this approach will work with prerender.io

### LINKED ISSUE #10: SEO - Prerender

### PR DESCRIPTION
**Pull request by adamduren**
Once finished this will resolve #10 allowing search engines and crawlers to correctly index the pages they visit.

- [x] Move off of firebase hosting
- [x] Setup SSL on new server
- [x] Configure prerender
- [x] Create component to insert the correct meta on job detail

### CODE REVIEW

**Comment by zacharytamas** [comment]
Does this actually work? This is kind of mind bending.

**Comment by adamduren** [comment]
Yep. You can see it at http://polygigs.adamduren.com

**Comment by adamduren** [comment]
I was inspired by your old `page-title` tag I found on some random Google Forum.

**Comment by adamduren** [comment]
I'd like to circle back with making a component that will generate all of our needed meta.

File: src/components/gig-detail.html [hunk 1]
```diff
@@ -54,6 +55,13 @@
   </style>
 
   <template is="dom-if" if="[[gig]]">
+    <dynamic-meta id-key="name" id-value="description" content="[[gig.posterCompany]] in [[gig.location]] is hiring for [[gig.title]]"></dynamic-meta>
```
Comment (adamduren) [inline src/components/gig-detail.html]: I want to create another component maybe called `social-meta` which accepts a title and description and encapsulates all the `dynamic-meta`'s to keep this from being redundant. This isn't even all of them but I wanted to get something out there.

**Comment by adamduren** [comment]
I think this is pretty much done for the initial purpose.

**Comment by zacharytamas** [comment]
I had already reviewed this but didn't explicitly approve it. 👍

**Pull request merged**